uaclient/cli/config.py

```diff
@@ -43,7 +45,7 @@ def action_config_show(args, *, cfg, **kwargs):
             )
         print(
             "{key} {value}".format(
-                key=args.key, value=getattr(cfg, args.key, None)
+                key=args.key, value=getattr(pro_config, args.key, None)
```

Comment:
We can refactor this function more.
First turn `pro_config` into a dict with `.to_dict(keep_none=True)`.
Instead of checking if the key is in `UA_CONFIGURABLE_KEYS`, check if it's in the dict.
Instead of using `getattr`, use `.get` on the dict.

Reply:
Do we want to implement these changes?

Reply:
I remember we decided to keep it the same since the pro_config variable dict does not have the `apt_http_proxy` and `apt_https_proxy` values.

We can keep the pro_config rename for the variable though since we are now calling the api for those values?@renanrodrigo

Reply:
whatever you have decided - I asked because there was a thumbs up and no conclusion in the thread. I'm fine